packages/parrot-devtools/src/app/components/Settings.jsx

```diff
@@ -23,6 +23,7 @@ const Settings = ({ url, setUrl }) => (
     <div className="pad-1">
       <h1 className="heading-5 pad-b">Settings</h1>
       <div className="form-group">
+        {/* eslint-disable-next-line jsx-a11y/label-has-associated-control */}
```

Comment:
Stick `name="url"` on the `input` so we don't have to write eslint exceptions?

Reply:
I tried this, it still gave me the lint error..